public/v4/apps/photoprism.yml

```diff
@@ -1,23 +1,119 @@
 captainVersion: 4
 services:
     $$cap_appname:
-        image: photoprism/photoprism:$$cap_photoprism_version
+        image: "photoprism/photoprism:$$app_version"
+        depends_on:
+            - $$cap_appname-db
+        environment:
+            PHOTOPRISM_ADMIN_PASSWORD: $$app_admin_password
+            PHOTOPRISM_AUTH_MODE: password
+            PHOTOPRISM_SITE_URL: "https://$$cap_appname.$$cap_root_domain"
+            PHOTOPRISM_ORIGINALS_LIMIT: $$app_upload_size_limit
+            PHOTOPRISM_HTTP_COMPRESSION: gzip
+            PHOTOPRISM_UPLOAD_NSFW: "true"
+            PHOTOPRISM_DATABASE_DRIVER: mysql
+            PHOTOPRISM_DATABASE_SERVER: srv-captain--$$cap_appname-db
+            PHOTOPRISM_DATABASE_NAME: photoprism
+            PHOTOPRISM_DATABASE_USER: photoprism
+            PHOTOPRISM_DATABASE_PASSWORD: $$db_password
         volumes:
-            - $$cap_appname-data:/photoprism/originals/
-        restart: always
+            - "$$cap_appname-originals:/photoprism/originals"
+            - "$$cap_appname-storage:/photoprism/storage"
+        working_dir: /photoprism
+        labels:
+            $$cap_appname-ofelia.enabled: $$ofelia_enabled
+            $$cap_appname-ofelia.job-exec.photoprism_index.schedule: "$$ofelia_index_schedule"
+            $$cap_appname-ofelia.job-exec.photoprism_index.command: "photoprism index --cleanup"
         caproverExtra:
-            containerHttpPort: '2342'
+            containerHttpPort: "2342"
+    $$cap_appname-db:
+        restart: unless-stopped
+        image: "mariadb:$$db_version"
+        command: >-
```

Comment:
This is an unsupported property. If it's important, use a different way to override the command. See this for example:
https://github.com/caprover/one-click-apps/blob/master/public/v4/apps/redis.yml#L10-L12